test/e2e/e2e_test.go

```diff
@@ -73,12 +76,15 @@ var _ = Describe("Workload cluster creation", func() {
 			}, result)
 			cluster = result.Cluster
 			targetCluster = bootstrapClusterProxy.GetWorkloadCluster(ctx, namespace, clusterName)
-			remediation()
```

Comment:
Don't forget to put remediation() back to the code when the PR goes from draft to normal state.